packages/sdk/src/types/runtime/assets/material.ts

```diff
@@ -11,23 +11,7 @@ import { Color3, Color4 } from '../../../math';
  */
 export interface MaterialLike {
     /** The base color of this material. */
-    color: Color4;
-    /** The texture ID assigned to this material, if any. */
-    textureId?: string;
-    /**

```

Comment:
Would be great to support metallic and roughness, at least. Is this still planned?

Reply:
Not technically. Our shader is vertex-lit, so reflection parameters are entirely unused. Don't want to get users excited over nothing.